claudedocs/image-optimization-guide.md

```
@@ -0,0 +1,201 @@
+<Image
+  src={imageUrl}
+  alt={title}
+  fill
+  className="object-cover"
+  priority  // ← Critical for above-fold images
+/>
+```
+
+## Critical Next Steps
+
+### 1. Compress Source Images (URGENT)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)